src/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -1057,31 +1084,36 @@ public byte[] ToByteArray()
         /// </summary>
         /// <param name="destination">The destination span to which the resulting bytes should be written.</param>
         /// <param name="bytesWritten">The number of bytes written to <paramref name="destination"/>.</param>
+        /// <param name="isUnsigned">Whether or not an unsigned encoding is to be used</param>
+        /// <param name="isBigEndian">Whether or not to write the bytes in a big-endian byte order</param>
         /// <returns>true if the bytes fit in <see cref="destination"/>; false if not all bytes could be written due to lack of space.</returns>
-        public bool TryWriteBytes(Span<byte> destination, out int bytesWritten)
+        /// <exception cref="OverflowException">If <paramref name="isUnsigned"/> is <c>true</c> and <see cref="Sign"/> is negative.</exception>
+        public bool TryWriteBytes(Span<byte> destination, out int bytesWritten, bool isUnsigned=false, bool isBigEndian=false)
         {
             bytesWritten = 0;
-            return TryGetBytes(GetBytesMode.Span, destination, ref bytesWritten) != null;
+            return TryGetBytes(GetBytesMode.Span, destination, isUnsigned, isBigEndian, ref bytesWritten) != null;
         }
 
-        /// <summary>Gets the number of bytes that will be output by <see cref="ToByteArray"/> and <see cref="TryWriteBytes(Span{byte}, out int)"/>.</summary>
+        /// <summary>Gets the number of bytes that will be output by <see cref="ToByteArray(bool, bool)"/> and <see cref="TryWriteBytes(Span{byte}, out int, bool, bool)"/>.</summary>
         /// <returns>The number of bytes.</returns>
-        public int GetByteCount()
+        public int GetByteCount(bool isUnsigned=false)
         {
             int count = 0;
-            TryGetBytes(GetBytesMode.Count, default(Span<byte>), ref count);
+            TryGetBytes(GetBytesMode.Count, default(Span<byte>), isUnsigned, false, ref count);
```

Comment:
Nit: name the false arg?  C# 7.2 allows that without naming everything after it.